Gemfile.lock

```diff
@@ -358,6 +358,7 @@ GEM
     mysql2 (0.4.9-x64-mingw32)
     mysql2 (0.4.9-x86-mingw32)
     nio4r (2.1.0)
+    nio4r (2.1.0-java)
```

Comment:
Is this change necessary for this pull request?

Reply:
This is unnecessary 💦  I removed it.